.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -1,3 +1,18 @@
+/**
```

Comment:
If you use mybatis-parent 31-SNAPSHOT, you can prevent adding license header on `MavenWrapperDownloader.java` like https://github.com/mybatis/mybatis-3/commit/b1f9387540c66ffc2188a35fda99c02784a95184#diff-600376dffeb79835ede4a0b285078036.

Thanks.

Reply:
Thanks for letting me know!